rust_snuba/src/consumer.rs

```diff
@@ -132,6 +132,9 @@ pub fn consumer_impl(
 
     let logical_topic_name = consumer_config.raw_topic.logical_topic_name;
 
+    let concurrency = ConcurrencyConfig::new(concurrency);
+    let clickhouse_concurrency = ConcurrencyConfig::with_runtime(2, concurrency.handle());
```

Comment:
if we used the existing runtime for clickhouse, we'd limit the CPU usage we could use for processing pool, and it turns out that passing `concurrency` to `with_runtime` actually has no effect. this is bad because we want to limit the amount of concurrency we use against clickhouse

Reply:
I reverted this change. Now the normal processor, clickhouse and dlq get their own runtimes when initializing the consumer, but still in a way that does not wreck the whole "run benchmarks in profiler" usecase.